webpack.config.js

```diff
@@ -113,7 +113,7 @@ const webpackConfig = ( mode ) => {
 				'googlesitekit-admin': './assets/js/googlesitekit-admin.js',
 				'googlesitekit-module': './assets/js/googlesitekit-module.js',
 				// Needed to test if a browser extension blocks this by naming convention.
-				adsense: './assets/js/adsense.js',
+				'dfp.js.ads': './assets/js/dfp.js.ads.js',
```

Comment:
I think it makes more sense to use use a pattern that matches AdSense as much as possible rather than a service we're not using.

I found that giving the script a name of `pagead2.ads.js` was also detected by AdBlock as well as another popular adblocker AdGuard. `pagead2` comes from the URL we use for loading `adsbygoogle.js` (unfortunately this is not detected).

The various patterns used by AdBlock can be found here:
https://easylist-downloads.adblockplus.org/easylist.txt
There are a few other lists it uses but those seem to be for exceptions or other rules.
Some of these patterns also seem to be matching for CSS classes or other selectors, not just URL fragments.

Reply:
SGTM, will change it to use `pagead2.ads.js`.